sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPSensorTest.java

```diff
@@ -449,6 +449,29 @@ public void should_disable_unnecessary_features_for_sonarlint() throws Exception
     assertThat(context.referencesForSymbolAt(mainFileKey, 6, 7)).isNotNull();
   }
 
+  @Test
+  public void no_measures_for_test_files() {
+    checkFactory = new CheckFactory(new ActiveRulesBuilder()
+      .addRule(newActiveRule("S5779"))
+      .build());
+    DefaultInputFile testFile = TestInputFileBuilder.create("moduleKey", "src/AppTest.php")
+      .setModuleBaseDir(context.fileSystem().baseDirPath())
+      .setType(InputFile.Type.TEST)
+      .setLanguage(Php.KEY)
+      .setContents("<?php echo $foo;\necho$bar;")
+      .build();
+
+    String testFileKey = testFile.key();
+
+    context.fileSystem().add(testFile);
+    context.setRuntime(NOT_SONARLINT_RUNTIME);
+
+    createSensor().execute(context);
+
+    assertThat(context.measure(testFileKey, CoreMetrics.NCLOC)).isNull();
```

Comment:
I think it's too easy for this test to be green for a bad reason. One way to change that is to have a test file where we expect to raise an issue: we should check that we have an issue and that NCLOC is null.